scss/components/_activity-panel.scss

```diff
@@ -89,6 +90,7 @@
 
 .civicase__activity-panel__core_container {
   min-height: 200px;
+  position: static !important;
```

Comment:
why the important?

Reply:
This panel have a inline `relative` positioning, which is coming from Core(as its a core screen). Thats why `!important` is necessary.